src/components/calcite-action/resources.ts

```diff
@@ -8,3 +8,5 @@ export const CSS = {
   slotContainerHidden: "slot-container--hidden",
   textContainer: "text-container"
 };
+
+export type TextDisplay = "hidden" | "visible" | "interactive";
```

Comment:
For consistency, could you move this to `interfaces.ts`? We've been using it for shared interfaces and types.